owl/SOMA-ACT.owl

```diff
@@ -379,6 +407,18 @@ Deduction is often explained as starting from the &quot;general&quot; (some prop
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#Delivering -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#Delivering">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Actuating"/>
+        <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Closing"/>
+        <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Fetching"/>
+        <owl:disjointWith rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Opening"/>
```

Comment:
Is this a complete list? I wonder because above in Closing there are more disjoint classes e.g. Squeezing.
This is because squeezing is not necessary an actuation? Like when I sit on a pillow it would be squeezing but no actuation?

Reply:
No, initially my idea was that some of these tasks might coincide (e.g., a Lifting might be Delivering). However, on second thought maybe it's better not to have this. If needed, Tasks that are disjoint can nonetheless still classify the same Action if we needed them to. So I made it such that the disjointWith lists are against all other concepts in Actuating.

Reply:
Well, being disjoint does not mean that it is not allowed to classify the same event, right? Tasks do not really coincidence at all, only the events that execute them

Reply:
Exactly. So the tasks can be of disjoint classes, and still classify the same event if that same event enacts two tasks somehow.